src/devices/Device-Index.md

```diff
@@ -119,7 +119,7 @@
 * [TCA954X - TCA954X Low-Voltage Multi-Channel I2C Switch with Reset](Tca954x/README.md)
 * [TCS3472x Sensors](Tcs3472x/README.md)
 * [TLC1543 - 10-bit ADC with 11 input channels](Tlc1543/README.md)
-* [TM1637 - Segment Display](Tm1637/README.md)
+* [TM1637 - Segment Display](Tm16xx/README.md)
```

Comment:
@pgrawehr  please remove this file, the changes are done automatically. And it's the reason the build fails!

Reply:
Well, "automatically" as in "when somebody manually runs the script", yes...